### PR TITLE
Clippy41

### PR DESCRIPTION
**Pull request by jaimetomares**
This is a new branch with error handling

### CODE REVIEW

File: api/views.py [hunk 1]
```diff
@@ -1,6 +1,96 @@
...
+from nltk.stem.snowball import SnowballStemmer
+from PIL import Image
+pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract.exe'
 
 def consume_file(request):
     if request.method == 'POST':
         print(request.FILES['file'])
-        return HttpResponse(request.FILES['file'].name)
\ No newline at end of file
+        return HttpResponse(request.FILES['file'].name)
```
Comment (AvendoOT) [inline api/views.py:20]: RETURN!!!

File: api/views.py [hunk 2]
```diff
@@ -1,6 +1,96 @@
...
+    
+        extension = request.FILES['file'].name.split(".")
```
Comment (AvendoOT) [inline api/views.py:22]: Write explanation for every line of code in this file

**Review by AvendoOT — CHANGES_REQUESTED** [review]

File: api/views.py [hunk 3]
```diff
@@ -1,6 +1,96 @@
...
+import pdf2image
+from nltk.corpus import stopwords
+from nltk.tokenize import word_tokenize, sent_tokenize
+from nltk.stem.snowball import SnowballStemmer
+from PIL import Image
+pytesseract.pytesseract.tesseract_cmd = r'C:\Program Files\Tesseract-OCR\tesseract.exe'
 
 def consume_file(request):
     if request.method == 'POST':
         print(request.FILES['file'])
```
Comment (IvanJuricic) [inline api/views.py:19]: Please add code for PdfFileReader so that you can extract page number

**Review by IvanJuricic — COMMENTED** [review]
Add *.pyc to your .gitignore file because those files don't need to be on our repo

File: api/views.py (hunk 2 above)
Comment (jaimetomares) [reply api/views.py:22]: You mean all views.py? (summarization program)

Comment (AvendoOT) [reply api/views.py:22]: Yes

**Pull request closed without merging**